doc/integrator/authentication.rst

```diff
@@ -1,6 +1,14 @@
 Authentication
 --------------
 
+Supported standard
+~~~~~~~~~~~~~~~~~~
+
+- `TOTP`: for the tow factor authentication (2fa), we can use with e.g. Google Authenticator.
```

Comment:
*two

Reply:
"two" as already stated by ochriste,
and there is a word missing in the second part of the sentence.
Suggestion: 
TOTP: for two-factor authentication, this can be used for example with Google Authenticator.